dockerfiles/zig-0.12.Dockerfile

```diff
@@ -18,15 +18,15 @@ RUN echo "@community http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /et
 # Update the package list and install Zig
 RUN apk add --no-cache zig@community=0.12.0-r0
 
-WORKDIR /app
+WORKDIR /src
```

Comment:
@glokta1 I recommend spending some time understanding what this actually does - the `/app` directory here is different from the `app` directory inside the starter code we're dealing with

If you're unfamiliar with dockerfiles, time to read up